libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -424,13 +424,6 @@ public void resumeCamera(Location location) {
     }
   }
 
-  @Override
-  public void updateNavigationMap(Location location) {
```

Comment:
Is there any use case in which the client would like to update the `NavigationMap` 🤔?